src/Parser.ts

```diff
@@ -42,44 +56,55 @@ export class Parser {
 
     for (let i = 0; i < lines.length; i += 1) {
       if (!startTime) {
-        startTime = this.parseTime(lines[i]);
+        startTime = Parser.parseTime(lines[i]);
         continue;
       }
 
-      if (!description || !ticketId) {
+      if (!ticketId && !description && lines[i].includes(":")) {
```

Comment:
Why did you add that constraint. That means, even if i got no description, i still have to add ':' after my ticket id. It worked without problems before...

Reply:
Without this check it recognises a "start time line" as ticket id.
```
#1.1.
12        <---- This "start time line"
  JIRA-123: some description
13
```

I found this location in the code to be only place to resolve the aliases because the validation of the time log entry is done while parsing.

Reply:
I would be happy to get your input on where we could resolve the aliases, without introducing breaking changes to the algorithm.